.gitlab-ci.yml

```diff
@@ -128,11 +128,12 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v4907599-0f65dfe
-  DATADOG_AGENT_BUILDERS: v4907602-55e258b
-  DATADOG_AGENT_WINBUILDIMAGES: v4907599-0f65dfe
-  DATADOG_AGENT_ARMBUILDIMAGES: v4907599-0f65dfe
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v4907599-0f65dfe
+  DATADOG_AGENT_BUILDIMAGES: v5407653-0eb225e
+  DATADOG_AGENT_BUILDERS: v5398909-05cc5dc
+  DATADOG_AGENT_WINBUILDIMAGES: v5407653-0eb225e
+  DATADOG_AGENT_ARMBUILDIMAGES: v5407653-0eb225e
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v5407653-0eb225e
+  BCC_VERSION: v0.19.0
```

Comment:
Hello ! `BCC_VERSION` was removed by https://github.com/DataDog/datadog-agent/pull/8779 as it is not used anymore